nca/NetworkConfig/TopologyObjectsFinder.py

```diff
@@ -37,7 +37,7 @@ def load_peer_from_calico_resource(self):
             self.add_eps_from_yaml(peer_code)
 
     def load_peer_from_k8s_live_cluster(self):
-        peer_code = yaml.load(CmdlineRunner.get_k8s_resources('pod'), Loader=yaml.CSafeLoader)
+        peer_code = yaml.load(CmdlineRunner.get_k8s_resources(['pod', 'serviceentry']), Loader=yaml.CSafeLoader)
```

Comment:
should split `serviceentry` to another function only for `istio` analysis?

Reply:
Done